libtiny_tui/src/text_field.rs

```diff
@@ -19,12 +20,18 @@ pub(crate) struct TextField {
     /// Cursor in currently shown line
     cursor: i32,
 
-    /// Horizontal scroll
-    scroll: i32,
-
     /// Width of the widget
     width: i32,
 
+    /// Max lines before turning into scroll mode
+    max_lines: i32,
+    /// Config value for text field wrapping
+    text_field_wrap: bool,
+    /// Whether or not scroll mode is currently on
+    scrolling_on: bool,
+    /// Amount of scroll in the input field
+    scroll: i32,
```

Comment:
So this value is only useful when `scrolling_on` is `true`, right? If that's the case than I think using `scroll: Option<i32>` and removing `scroll_on` might be better. If `scroll` is `None` then we know that scrolling is off. Otherwise the value gives us the scroll value.

Reply:
I think I can do that. Thanks for the tip!